lib/liquid/tags/doc.rb

```diff
@@ -31,21 +31,26 @@ module Liquid
   class Doc < Block
     NO_UNEXPECTED_ARGS = /\A\s*\z/
 
+    attr_reader :body
+
     def initialize(tag_name, markup, parse_context)
```

Comment:
Should we set `@body` to an empty string in `initialize`? It feels weird to have an attr_reader on something that doesn't exist until `parse` gets run.